documentation/asciidoc/topics/proc_restoring_cluster.adoc

```diff
@@ -42,13 +41,14 @@ include::yaml/backup_resources.yaml[]
 ----
 $ {oc_apply_cr} my-restore.yaml
 ----
-+
-A new pod joins the {brandname} cluster and restores state from the backup file.
-When the operation is complete, the pod leaves the cluster and logs the following message:
+
+.Verification
+
+. Check {brandname} logs for the following message:
```

Comment:
We should probably tell the user to check the `Restore` CR status as well. In the event of a successful restore, the `.status.phase` will be set to `Succeeded`.

Reply:
Is this true for `Backup` CR as well?

Reply:
Yep

Reply:
maybe we should create a new procedure for handling failed backup/restore operations?

Reply:
Do you mean document how to handle failures? Or implement something in the operator? I'm definitely in favour of the former.

Reply:
Exactly, we should document how to handle failures. I'll update shortly with a topic we can work through.